docs/installation/server-config/index.rst

```diff
@@ -180,6 +175,9 @@ server pool. With the default ``ksql.service.id``, the command topic would be ``
 the other hand, if you set ``ksql.service.id`` to ``production_deployment_``, the KSQL command topic will 
 be ``_confluent-ksql-production_deployment__command_topic``.
 
+By setting the ``ksql.service.id`` appropriately, you can have isolated pools of KSQL servers sharing the same underlying 
```

Comment:
how about `You can configure ``ksql.service.id`` so that your environment has isolated pools of KSQL servers sharing the same underlying Kafka cluster.`